src/schema/event-booking.json

```diff
@@ -155,6 +155,11 @@
             "category": "client",
             "description": "Refund option does not match one of the options we are expecting"
         },
+        {
+            "code": "SeatsUnavailable",
```

Comment:
I'm not sure we need to use the same error code from the different domain if the meaning is similar. we have the similar error in FlightBooking domain `SeatSelectionNotAvailable`

Reply:
Should we move `SeatSelectionNotAvailable` to Generic?

Reply:
It's not applicable for `MotorInsurance` or `VacationRental`, shall we have both `FlightBooking. SeatSelectionNotAvailable` and `EventBooking. SeatSelectionNotAvailable` ?